js_modules/dagster-ui/packages/ui-core/src/util/getJSONForKey.ts

```diff
@@ -1,11 +1,11 @@
 export function getJSONForKey(key: string) {
   let stored = undefined;
+  if (typeof localStorage === 'undefined') {
+    return undefined;
+  }
+
   try {
-    if (typeof window !== 'undefined') {
-      stored = window.localStorage.getItem(key);
-    } else {
-      stored = self.localStorage.getItem(key);
```

Comment:
As discussed in Slack, this probably errors since the WebWorker won't have `localStorage` available.